step-framework-core/src/main/java/step/core/async/AsyncProcessor.java

```diff
@@ -56,6 +56,8 @@ private void processQueue() {
             } catch (InterruptedException e) {
                 Thread.currentThread().interrupt(); // Restore interrupted status
                 break;
+            } catch (Throwable e){
+                logger.error("An exception occurred during the asynchronous processing an element of the queue.", e);
```

Comment:
Either "while asynchronously processing an element", or "during the asynchronous processing **of** an element" :-) -- I personally prefer the former.